### PR TITLE
Added Quarkus creation function

### LINKED ISSUE #80: create quarkus
As a user, I want to create projects using Quarkus.

### PR DESCRIPTION
**Pull request by veikkahonkanen**
Resolves #80

### CODE REVIEW

File: src/commands/new-proj/create-quarkus.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @param projName desired project name
+ * 
+ * @param kotlin true/false if using Kotlin
+ * 
+ * @param gradle true/false if using Gradle
+ * 
+ * @returns a command that is used to create the Quarkus project
+ */
+export const CreateQuarkus = async (projName : string, kotlin : boolean, gradle : boolean) : Promise<string> => {
+  const orgName = "fi.metatavu";
```
Comment (eetupur) [inline src/commands/new-proj/create-quarkus.ts]: shouldnt this be in env?

File: src/commands/new-proj/create-quarkus.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  let kotlinS : string = null;
```
Comment (eetupur) [inline src/commands/new-proj/create-quarkus.ts]: more readable, do same for kotlinS
```suggestion
  let kotlinString : string = null;
```

**Review by eetupur — CHANGES_REQUESTED** [review]
suggesting better naming

**Review by eetupur — APPROVED** [review]

File: src/commands/new-proj/create-quarkus.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * Constructs creation command for Quarkus projects
+ * @param projName desired project name
+ * 
+ * @param kotlin true/false if using Kotlin
+ * 
+ * @param gradle true/false if using Gradle
+ * 
+ * @returns a command that is used to create the Quarkus project
+ */
+export const CreateQuarkus = async (projName : string, kotlin : boolean, gradle : boolean) : Promise<string> => {
```
Comment (jarinykanen) [inline src/commands/new-proj/create-quarkus.ts]:
```suggestion
/**
 * Constructs creation command for Quarkus projects
 *
 * @param projName desired project name
 * @param kotlin true/false if using Kotlin
 * @param gradle true/false if using Gradle
 * 
 * @returns a command that is used to create the Quarkus project
 */
export const CreateQuarkus = async (projName: string, kotlin: boolean, gradle: boolean): Promise<string> => {
```

File: src/commands/new-proj/create-quarkus.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  const { ORGANIZATION } = process.env;
+  let kotlinString : string = null;
+  kotlin ? kotlinString = ", kotlin" : kotlinString = "";
+  let gradleString : string = null;
+  gradle ? gradleString = "\
```
Comment (jarinykanen) [inline src/commands/new-proj/create-quarkus.ts]:
```suggestion
  const kotlinString: string =  kotlin ? ", kotlin" : "";
  const gradleString: string = gradle ? gradleString = "\
  -DbuildTool=gradle" : "";
```

**Review by jarinykanen — CHANGES_REQUESTED** [review]

**Review by eetupur — APPROVED** [review]

**Review by jarinykanen — APPROVED** [review]

**Pull request merged**